code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -8,7 +8,7 @@
 	w_class = 3
 	origin_tech = "biotech=3"
 	var/braintype = "Cyborg"
-
+	var/obj/item/device/radio/radio = null //Let's give it a radio.

```

Comment:
We have a `radio/integrated` type for this that the cyborg mob uses... You might want to check that out.